modules/ui/sections/background_list.js

```diff
@@ -164,14 +97,41 @@ export function uiSectionBackgroundList(context) {
         });
     }
 
-    function drawListItems(layerList, type, change, filter) {
-        var sources = context.background()
+    function categoryMapping(layer) {
+        var categoryWeights = {
+            'photo': categoryGroups.photo,
```

Comment:
> not all imagery is currently properly categorized in ELI – see for example that "USGS Topographic Maps" is falsely listed under "aerial imagery" (the category used as fallback in this PR if a layer has no category specified). Using the categories in iD might help getting more layers properly categorized over time.

Indeed, fixed in osmlab/editor-layer-index@4c1aeb39c679dc7bc6e65a45ebea63851eea6d7e. 😄